modules/fb_instant_articles_display/fb_instant_articles_display.module

```diff
@@ -5,6 +5,132 @@
  * Hook implementations for Facebook Instant Articles Display module.
  */
 
+/**
+ * Load all Field module hooks for Facebook Instant Articles Display module.
+ */
+require_once __DIR__ . '/includes/field.inc';
+
+/**
+ * Implements hook_entity_load().
+ */
+function fb_instant_articles_display_entity_load($entities, $type) {
+  $enabled_entity_types = fb_instant_articles_display_get_article_entity_types();
+  foreach ($entities as $entity) {
+    // Only proceed if the entity is of the configured entity type and bundle.
+    list($id, $vid, $bundle) = entity_extract_ids($type, $entity);
+    if (!in_array($type, array_keys($enabled_entity_types))
+      || !in_array($bundle, array_keys($enabled_entity_types[$type]))) {
+      return FALSE;
+    }
+
+    // We add Entity-specific information to the ArticleWrapper object, and
+    // attach it to the Entity for use in later hooks. Note that we don't store
+    // the InstantArticle object on the node directly, because our Singleton
+    // wrapper + Drupal hook class pattern relies on invoking
+    // hook_fb_instant_articles_article_alter() in ArticleWrapper->getArticle(),
```

Comment:
Comment needs to be updated, since the alter happens in `ArticleWrapper::__construct` now.